contributing-guides/style-guide.md

```diff
@@ -76,3 +76,18 @@ Use backticks on the following:
 1. Paths, ex. `package.json`, `/etc/package.json`.
 2. Extensions, ex. `.dll`.
 3. Commands, ex. `ls`.
+
+## Serial Comma
+When declaring a list of items with 3 or more elements, use a [serial comma](https://en.wikipedia.org/wiki/Serial_comma), also known as the Oxford comma.
```

Comment:
its also known as the Harvard Comma.

Reply:
`elements` works better but what do you think about `items`. I have no strong opinion on `items` so I don't care.

Reply:
Any opinions on if we should mention Oxford/Harvard comma?

```suggestion
When declaring a list of items with 3 or more elements, use a [serial comma](https://en.wikipedia.org/wiki/Serial_comma), also known as the Oxford comma or Harvard comma.
```

In general, it could be better to reword this too, I kinda hate it. 🤔 

```suggestion
When declaring a list of 3 or more items, use a [serial comma](https://en.wikipedia.org/wiki/Serial_comma). (also known as an Oxford comma or Harvard comma)
```

* a list of items with 3 or more elements -> a list of 3 or more items
* change how we say "also known as"

Reply:
I agree with your 2nd change however thinking about it, I think the `also known as the Oxford comma` is pointless as anyone that clicks on the wiki link will find that information anyway

Reply:
The reason I think the Oxford comma part might be relevant is because back in school (and informal conversation) it's primarily referred to as the Oxford comma.

My partner also only knew it as an "Oxford comma" until now.

I have no problem removing it if others agree, this comment isn't indicating a preference. Only justifying why I put it there in the first place.

Reply:
Might that just be a British name, since Oxford University.........

Reply:
My partner is from Belgium. (and that's where I live)
I have no doubt the reason I call it "Oxford comma" is because I'm British, but most of my non-Brit friends also call it that.

Reply:
For a more objective perspective: <https://trends.google.com/trends/explore?geo=US&q=Oxford%20comma,Harvard%20comma,Serial%20comma>

Reply:
@marchersimon Nice idea, I'll keep that in mind for future.